storage/hasherstore.go

```diff
@@ -19,27 +19,30 @@ package storage
 import (
 	"context"
 	"fmt"
-	"sync/atomic"
+	"sync"
 
 	"github.com/ethersphere/swarm/chunk"
 	"github.com/ethersphere/swarm/storage/encryption"
 	"golang.org/x/crypto/sha3"
 )
 
+const (
+	noOfStorageWorkers = 150
```

Comment:
Data chunks are batched as 128 to form a tree chunk. If we want this storage to be in sync with the chunker speed, its better to have all 128 data chunks pushed using a separate go-routine. I added few more to take care of the Tree chunks. This will allow both the chunker and storage to move in steps of 128.